dashboard/config/linux/bits/chromeos-5.15.yml

```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+kernel:
+ repo: https://chromium.googlesource.com/chromiumos/third_party/kernel
+ tag: 0faa8c9f9dc4
+
+shell:
+ - chromeos/scripts/prepareconfig chromiumos-x86_64 ${BUILDDIR}/.config
+ - make olddefconfig
+
+config:
+ # DRM_I915 is broken in 5.4 with:
```

Comment:
Is it still broken with 5.15?

Reply:
Checked w/o this line. Kernel build is ok.